stable/sonarqube/templates/copy-plugins.yaml

```diff
@@ -25,6 +25,4 @@ data:
         done
       done
       cp /opt/sonarqube/extensions/plugins/tmp/*.jar /opt/sonarqube/extensions/plugins/
```

Comment:
Could you look for other occurrences of `/opt/sonarqube/` and also use the new variable there?
Otherwise we fix the run.sh script, but for instances installation of plugins won't work.